src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StorageTest {
+    private final DateTimeFormatter parse_format = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm a");
+    @Test
+    public void testCreateFile() throws DukeException {
+        new Storage("testing/test.txt");
+
+        assertTrue(Files.exists(Paths.get("testing/test.txt")), "File should be created");
+    }
+
+    @Test
+    public void testWriteFile() throws DukeException {
+        Storage storage = new Storage("testing/test.txt");
```

Comment:
You could consider having "testing/test.txt" as a constant string declared before the test cases run to save some lines of code but overall good job 👍